test/jdk/java/awt/8368882/FallbackFontNPE.sh

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2025, Azul Systems, Inc. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+
+# @test
+# @bug 8368882
+# @compile FallbackFontNPE.java
+# @requires os.family == "windows"
+# @run shell/timeout=400 FallbackFontNPE.sh
+# @summary Check if PhysicalFont.getMapper() doesn't throw NPE
+
+if [ -z "${TESTSRC}" ]; then
+  echo "TESTSRC undefined: defaulting to ."
+  TESTSRC=.
+fi
+
+if [ -z "${TESTCLASSES}" ]; then
+  echo "TESTCLASSES undefined: defaulting to ."
+  TESTCLASSES=.
+fi
+
+if [ -z "${TESTJAVA}" ]; then
+  echo "TESTJAVA undefined: can't continue."
+  exit 1
+fi
+
+cd ${TESTSRC}
+
+echo Setup EUDC.tte font
+reg_eudc_1252="HKCU\\EUDC\\1252"
+echo ...Write ${reg_eudc_1252} record
+old_reg_record=""
+if Reg QUERY "${reg_eudc_1252}"; then
+   old_reg_record=$(cygpath -m ${TESTCLASSES}/eudc_1252.reg)
+   Reg EXPORT "${reg_eudc_1252}" ${old_reg_record} /y
+   Reg DELETE "${reg_eudc_1252}" /va /f
+fi
+Reg import eudc.reg
```

Comment:
Dont think we are allowed to push binary files..Cant we check using 
`-Duser.language `or 
`new Locale(ja, JP)`

Reply:
The test doesn't change locale, and CLI settings doesn't help since native AWT library uses system [GetSystemDefaultLangID](https://github.com/openjdk/jdk/blob/0a97bef840f8799313a1a55a65d9334e09cc1cf4/src/java.desktop/windows/native/libawt/windows/awt_Win32GraphicsEnv.cpp#L238) call.

And unfortunately without this file whole test is useless.